website/docs/install-config/configuration/configuration.mdx

```diff
@@ -354,10 +354,6 @@ Configure how long reputation scores should be saved for in seconds. Note that t
 
 Defaults to `86400`.
 
-### `AUTHENTIK_SESSION_STORAGE`:ak-version[2024.4]
-
-Configure if the sessions are stored in the cache or the database. Defaults to `db`. Allowed values are `cache` and `db`. Note that changing this value will invalidate all previous sessions.
-
```

Comment:
I don't think we want to remove this, because as Jens pointed out, people who are still running a pre-2025.4 version will want to know this process.

Haha we need a way to show a badge "range"...

Reply:
I'm thinking about how to indicate that after 2025.4, this option is deprecated... Ah. Let's add a :::info saying exactly that.